### ISSUE TITLE
[error]: Fail to fetch HA Version (Response code 403 (Forbidden))

### ISSUE THREAD

**Issue by bachy** [Sebclem/hassio-nextcloud-backup#30]
Hi,

since 2020/11/09 (date of my last NC backup), i can't launch backup

**Describe the bug**
Can't launch backup
with error
```
[error]: Fail to fetch HA Version (Response code 403 (Forbidden))
```
full log
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 10-run: applying... 
[fix-attrs.d] 10-run: exited 0.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[22:18:31] INFO: Starting Node...
> nexcloud-backup@0.8.0 start /opt/nextcloud_backup
> node --inspect=0.0.0.0:9226 ./bin/www 
Debugger listening on ws://0.0.0.0:9226/4ce61aef-42c0-467d-8d2f-b10a0cb95b6f
For help, see: https://nodejs.org/en/docs/inspector
[2020-11-14 23:18:37] [info]: Satus : Go !
[2020-11-14 23:18:37] [info]: Initilizing and checking webdav client...
[2020-11-14 23:18:37] [info]: Starting auto clean cron...
[2020-11-14 23:18:38] [info]: Starting Cron...
[2020-11-14 23:18:38] [info]: Hassio API : Go !
[2020-11-14 23:18:39] [debug]: Nextcloud connection:  Success !
[2020-11-14 23:18:39] [debug]: Path /Hassio Backup/ already exist.
[2020-11-14 23:18:40] [info]: Nextcloud connection : Go !
[2020-11-14 23:19:27] [error]: Fail to fetch HA Version (Response code 403 (Forbidden))
```
**Expected behavior**
do backup

**System information**
- Home Assistant Version : HassOS 4.16
- Home Assistant Supervisor Version : 2020.11.0
- Nextcloud Version : NC is not the probleme
- Addon version : 0.11.0

```
arch | armv7l
-- | --
chassis | embedded
dev | false
docker | true
docker_version | 19.03.12
hassio | true
host_os | HassOS 4.16
installation_type | Home Assistant OS
os_name | Linux
os_version | 4.19.127-v7
python_version | 3.8.6
supervisor | 2020.11.0
timezone | Europe/Paris
version | 0.117.6
virtualenv | false
```
**More context**
rpi 3 b+
recorder use mariadb
data partition is on a 120G usb ssd 

core snapshot is working
```
20-11-14 22:31:00 INFO (MainThread) [supervisor.snapshots] Found 1 snapshot files
20-11-14 22:31:38 INFO (MainThread) [supervisor.snapshots] Creating new full-snapshot with slug e3a91ecc
20-11-14 22:31:38 INFO (MainThread) [supervisor.snapshots] Snapshotting e3a91ecc store Add-ons
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_nodered
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_nodered
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_samba
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_samba
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_ssh
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_ssh
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_mosquitto
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon core_mosquitto
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on 229cc4d7_nextcloud_backup
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon 229cc4d7_nextcloud_backup
20-11-14 22:31:38 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_influxdb
20-11-14 22:34:52 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_influxdb
20-11-14 22:34:52 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on a0d7b954_grafana
20-11-14 22:34:52 INFO (MainThread) [supervisor.addons.addon] Finish snapshot for addon a0d7b954_grafana
20-11-14 22:34:52 INFO (MainThread) [supervisor.addons.addon] Building snapshot for add-on core_mariadb
...
```
found that in logs
```
20-11-14 22:19:27 WARNING (MainThread) [supervisor.api.security] /core/info no role for 229cc4d7_nextcloud_backup
20-11-14 22:19:27 ERROR (MainThread) [supervisor.api.security] Invalid token for access /core/info
```

**Comment by poudenes** [comment]
Have same issue...

**Comment by Sebclem** [comment]
Hi,

It's normaly fixed with [V0.11.1](https://github.com/Sebclem/hassio-nextcloud-backup/releases/tag/0.11.1). 
If the update not showing up, you can force update check by going to Supervisor -> Addon Store -> 3 dots on top right -> Reload

**Comment by bachy** [comment]
yes, it's working now, thanks for the fix, and for your addon !
